src/site/content/en/blog/youtube-media-capabilities/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: YouTube enhances UX with Media Capabilities Web API
```

Comment:
```suggestion
title: YouTube enhances UX with Media Capabilities API
```

I was wondering if it's possible to drop the word "Web" from the title because I think it may be implied.

Reply:
Done.